packages/api-nest/src/repositories/library-item.repository.ts

```
@@ -155,6 +174,32 @@ export class LibraryItemRepository implements ILibraryItemRepository {
     return this.repository.create(data)
   }
 
+  /**
+   * Update specific fields of a library item
+   */
```

ESLint

[error] Missing JSDoc @returns for function.

(valid-jsdoc)